mmcif/__init__.py

```diff
@@ -2,6 +2,6 @@
 __author__ = "John Westbrook"
 __email__ = "john.westbrook@rcsb.org"
 __license__ = "Apache 2.0"
-__version__ = "0.85.1"
+__version__ = "0.86"
```

Comment:
I think we want to stick with major.minor.patch versioning. How about `0.86.0`?

Reply:
I am ok with making the change. ChangeLog would need updating.

Reply:
See new commit